ast/field.go

```diff
@@ -24,7 +24,7 @@ func (f *Field) Override(key string, def string) string {
 func (f *Field) parseOverrides() {
 	f.overridesParsed = true
 	f.overrides = make(map[string]string)
-	fields := strings.Split(f.Tag, ",")
+	fields := strings.Split(f.Tag, " ")
```

Comment:
I believe these are supposed to be space delimited, but I didn't spend a lot of time validating that belief.